Assignment3.txt

```diff
@@ -0,0 +1,9 @@
+Albert Zhang
+1)Not sure if this is right please correct me if I'm wrong.
+	1. O(n) = n
+	2. O(n) = n + n^2
+	3. O(n) = log(n)
```

Comment:
The first for loop gives a time complexity of n since you iterate through n elements. The second for loop would normally also be n, but since it multiplies the j by two each time, that means you're eliminating half the number of elements each time - this causes a time complexity of log (n). And since they're nested, you multiply the two with each other so that the time complexity is O(nlog(n)).